pom.xml

```diff
@@ -94,18 +89,12 @@
 
   <profiles>
     <profile>
-      <id>release</id>
+      <id>jenkins-release</id>
       <build>
         <plugins>
           <plugin>
             <artifactId>maven-source-plugin</artifactId>
             <executions>
-              <execution>
-                <id>attach-sources</id>
-                <goals>
-                  <goal>jar-no-fork</goal>
-                </goals>
-              </execution>
```

Comment:
The parent POM's `jenkins-release` profile includes this logic as of jenkinsci/pom#296. However, the logic to attach the test sources is still necessary. I tested this by running `mvn clean verify -Pjenkins-release` and verified that the sources and test sources still got generated.